training/model_management/components/import_model/spec.yaml

```diff
@@ -4,14 +4,18 @@ type: pipeline
 name: import_model
 display_name: Import model
 description: Import a model into a workspace or a registry
-version: 0.0.5
+version: 0.0.6
 
 # Pipeline inputs
 inputs:
   # pipeline specific compute
-  compute:
+  compute_common:
```

Comment:
What is the reason to rename this input?

Reply:
Reverted